src/screens/VisualTests/VisualTests.stories.tsx

```diff
@@ -238,6 +255,97 @@ export const EmptyBranchLocalBuildCapturing = {
   },
 } satisfies Story;
 
+// There is a build, but this story is new (not on the build at all)
+export const StoryAddedNotInBuild: Story = {
+  parameters: {
+    ...withBuilds({ selectedBuild: newStoryNoTests }),
+    ...withFigmaDesign(
+      "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=1898-562751&mode=design&t=ciag0nGKx2OGmoSR-4"
+    ),
+  },
+};
+
+export const StoryAddedNotInBuildStarting: Story = {
+  parameters: {
+    ...withBuilds({
+      selectedBuild: newStoryNoTests,
+    }),
+    ...withFigmaDesign(
+      "https://www.figma.com/file/GFEbCgCVDtbZhngULbw2gP/Visual-testing-in-Storybook?type=design&node-id=1898-562751&mode=design&t=ciag0nGKx2OGmoSR-4"
+    ),
+  },
+  args: {
+    localBuildProgress: {
+      buildProgressPercentage: 1,
+      currentStep: "initialize",
+      stepProgress: {
+        ...INITIAL_BUILD_PAYLOAD.stepProgress,
+        initialize: { startedAt: Date.now() - 1000 },
+      },
+    },
+  },
+};
+
+export const StoryAddedNotInBuildCompletedLocalProgressIsOnSelectedBuild: Story = {
```

Comment:
I'm not sure I'm understanding this name (naming stories is sort of a challenge right now). 

If I am reading the story correctly, the current story is:
  - Not on the selected build
  - On the last build on branch?
  - and the local build is done?

Does this correspond to "New story, latest build on branch has it and has captured it" from [this comment?](https://github.com/chromaui/addon-visual-tests/pull/101#discussion_r1334858918)